submissions/ArthurGorbenko/html.css.practice/css/styles.css

```diff
@@ -0,0 +1,169 @@
+body{
+    font-family: arial,sans-serif;
+    font-size: 15px;
+}
+
+.container{
+    display: flex;
+    justify-content: flex-end;
+}
+
+div{
+    margin: 10px;
+}
+
+.content{
+    height: 500px;
+    width: 350px;
+    border: 1px solid #ccc;
+    float: right;
+    box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.15);
+    overflow-y: auto;
+    display:none;  
+}
+
+ul{
```

Comment:
Very good job.
But.
Avoid using styling by tags. It's ok for `<html>` and `<body>`, but not for other elements. Use classes or id's instead.